composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^5.6|^7.0",
         "doctrine/dbal": ">=2.4.5,<3.0.x-dev",
-        "phpcr/phpcr": "~2.1.2",
+        "phpcr/phpcr": "~2.1.3",
```

Comment:
Ok, you're right will close it. :)

Today I'm confused because I thought that I updated this constants: https://github.com/phpcr/phpcr/blob/master/src/PHPCR/PropertyInterface.php :o